src/commands/gameStatus.ts

```diff
@@ -0,0 +1,139 @@
+import { AccessLevel, Command, CommandError } from "./command";
+import { Message, MessageEmbed, MessageActionRow, MessageSelectMenu, MessageSelectOptionData } from "discord.js";
+import { Octokit } from "@octokit/rest";
+const octokit = new Octokit();
+
+/** This command is used to check the compatability of a game on Skyline through the Skyline Games List on GitHub */
```

Comment:
Minor typo. Should be "compatibility" rather then "compatability".

Reply:
fixed